.travis.yml

```diff
@@ -19,11 +18,42 @@ addons:
     packages:
       - docker-ce
 
-script: npm run check-package && npm run lint && npm run test
+env:
+  - DOCKER_COMPOSE_VERSION=1.25.5
+
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+install:
+  - npm i
+  # Build docker image
+  - docker build -t platform-test-suite:local .
+  # Install MN Bootstrap
+  - chmod +x ./scripts/get-gh-release-link
```

Comment:
You don’t need to change it everytime. Jsut set it locally and commit to git?

Reply:
https://stackoverflow.com/questions/40978921/how-to-add-chmod-permissions-to-file-in-git